modules/microshift-manifests-removal-scenario-rpm.adoc

```diff
@@ -15,16 +15,16 @@ Use the following procedure in the data removal scenario for RPM systems to comp
 +
 [source,terminal]
 ----
-$ sudo mkdir -p _<path_of_delete_directory>_ <1>
+$ sudo mkdir -p <path_of_delete_directory> <1>
 ----
-<1> Replace `_<path_of_delete_directory>_` with the path of the delete subdirectory, for example, `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
+<1> Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete`.
```

Comment:
Nit, but also out of scope - [serial/Oxford comma](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series) needed here:
```suggestion
<1> Replace `_<path_of_delete_directory>_` with one of the following valid directory paths: `/etc/microshift/manifests.d/delete`, `/etc/microshift/manifests/delete/`, `/usr/lib/microshift/manifests.d/delete`, or `/usr/lib/microshift/manifests/delete`.
```

Reply:
done